### DISCUSSION TITLE
Ukrainian translation

### DISCUSSION THREAD

**Discussion by matrunchyk** [vuejs/router#1770] [Ideas]
### What problem is this solving

There is no Ukrainian translation

### Proposed solution

Adding the Ukrainian translation

### Describe alternatives you've considered

Hello guys,

I'm one of maintainers of [vuejs-translations](https://github.com/vuejs-translations/docs-uk) team, and our team is looking forward to participating in Ukrainian translation of Vue-Router (as we just completed ua.vuejs.org translation).

I see the translation of vue-router is different than vuejs.org (no separate repo), so I created this issue just to understand our next steps.
Do you accept new PRs to this repo for new translations? If so, are there any specific requirements?

Thanks.

**Comment by posva** [comment]
Thanks a lot for the initiative! I need to sync with @Jinjiang to add instructions on how to add and sync translations to the docs

**Reply by matrunchyk** [reply]
Thank you! What we've done so far - just forked the repo and created `uk` folder inside docs folder, same way as you have for `zh`.
So once we've fully completed the translation I'll create a PR to merge our changes into this repo.

Please let me know if it fine by you and @Jinjiang.

**Reply by Jinjiang** [reply]
@posva speaking of the instruction, do you think it's a good idea to create a Wiki page for that? (currently, we didn't enable the wiki)

@matrunchyk before that, I think you can refer to this discussion as a dry-run: https://github.com/Jinjiang/router/pull/1. More examples: https://github.com/vuejs/router/pull/1726, https://github.com/vuejs/router/pull/1773

Thanks.

**Reply by matrunchyk** [reply]
@Jinjiang yeah that's how I expected to keep it in sync (basically we do almost the same for Vue docs.

**Reply by posva** [reply]
@Jinjiang I think that we can just add an `.md` or a section to the contributing guide in the `.github` folder

**Reply by Jinjiang** [reply]
Sure. Let me draft one first.

**Reply by Jinjiang** [reply]
Update: the PRs about the translation guide are here #1778 #1788
